typeDefinition/modifiers.d.ts

```diff
@@ -98,12 +98,22 @@ export interface WillMadeModifier<T> {
 	willBeMade(): T;
 }
 
+type RequestPropName = symbol | string;
+type RequestPropValue = symbol | string | number;
+type RequestPropComparator = symbol;
```

Comment:
Probably you should change this type to string, because we changed comparator constants from symbol to string, but I dont remember in which branch.

Reply:
That was done by Anton on sp-2 branch. We'll release it later then this one.